salt/metalk8s/roles/etcd/init.sls

```diff
@@ -1,2 +1,2 @@
-Deploy etcd member:
-  test.succeed_without_changes
+include:
+  - metalk8s.kubernetes.etcd
```

Comment:
Why wouldn't this include `node` when `master` includes it?

Reply:
True I can include `node`

Reply:
Have to: whilst `etcd` strictly-speaking only requires `kubelet`, and not necessarily a `kubelet` connecting to an API server (so, not `node`), in our deployments we *do* want said `kubelet` to talk to API server and such :)

Reply:
Hmm I will only include kubelet.standalone then and btw in our case `etcd` node will likely be `node` also so they will get a `kubelet` configured to talk with `apiserver` look better I think :)